### PR TITLE
backward compatibility for sync stream

### PR DESCRIPTION
**Pull request by sitaram-kalluri**
**- What I did** Provide backward compatibility for stream sync verb.

**- How I did it** Move stream sync to a new verb handler - sync:stream and have sync for regular sync process.

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: at_secondary/at_secondary_server/lib/src/verb/handler/sync_verb_handler.dart [hunk 1]
```diff
@@ -16,7 +16,7 @@ class SyncVerbHandler extends AbstractVerbHandler {
 
   @override
   bool accept(String command) =>
-      command.startsWith(getName(VerbEnum.sync) + ':');
+      command.startsWith(getName(VerbEnum.sync) + ':') && !command.contains('stream');
```
Comment (murali-shris) [inline at_secondary/at_secondary_server/lib/src/verb/handler/sync_verb_handler.dart]: !command.startsWith('sync:stream')

File: at_secondary/at_secondary_server/lib/src/verb/manager/response_handler_manager.dart [hunk 2]
```diff
@@ -40,8 +40,8 @@ class DefaultResponseHandlerManager implements ResponseHandlerManager {
       return _streamHandler;
     } else if (verb is NotifyAll) {
       return _notifyAllHandler;
-    } else if (verb is Sync) {
```
Comment (murali-shris) [inline at_secondary/at_secondary_server/lib/src/verb/manager/response_handler_manager.dart:43]: for sync..baseresponse handler will be used?

File: at_secondary/at_secondary_server/lib/src/verb/handler/sync_verb_handler.dart (hunk 1 above)
Comment (sitaram-kalluri) [reply at_secondary/at_secondary_server/lib/src/verb/handler/sync_verb_handler.dart]: Will change it to sync:stream

File: at_secondary/at_secondary_server/lib/src/verb/manager/response_handler_manager.dart (hunk 2 above)
Comment (sitaram-kalluri) [reply at_secondary/at_secondary_server/lib/src/verb/manager/response_handler_manager.dart:43]: Yes Murali, for sync BaseResponseHandler will used.

**Pull request merged**